src/server_manager/messages/en.json

```diff
@@ -66,6 +66,8 @@
   "error-do-account-info": "Failed to get DigitalOcean account information",
   "error-do-auth": "Authentication with DigitalOcean failed",
   "error-do-regions": "Failed to get list of available regions",
+  "error-do-limit": "Your DigitalOcean account has reached its limit of {num} Droplets.  You can request an increase at https://cloud.digitalocean.com/account/team/droplet_limit_increase",
+  "error-do-warning": "DigitalOcean warning: \"{message}\" ({detail})",
```

Comment:
The "status:" literal needs to be translated (inlined here), or be part of the untranslated warning string.
I'd favor putting the status in the untranslated warning for simplicity.

Reply:
OK, done